simulator/mips/mips_instr.h

```diff
@@ -206,11 +207,9 @@ class MIPSInstr
 
         /* info for branch misprediction unit */
         bool predicted_taken = false;     // Predicted direction
```

Comment:
Actually these two variables should be encapsulated into BPInterface since they are originated from BP.